src/components/Navbar/NavbarMobile.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { FiMenu } from 'react-icons/fi';
+import {
+  Button,
+  Flex,
+  Image,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Spacer,
+} from '@chakra-ui/react';
+import NavbarLinkMobile from './NavbarLinkMobile';
+import ProfileDropdown from './ProfileDropdown';
+
+import logo from '../../assets/OCHlogo.png';
+
+const NavbarMobile = ({ isAdmin }) => {
+  // TO DO: get profile image and name
+  const user = {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'user' is assigned a value but never used.